src/hooks/useIsNativePassInstalled.ts

```diff
@@ -0,0 +1,16 @@
+import { useState, useEffect } from 'react'
+import { Linking } from 'react-native'
+
+export const useIsNativePassInstalled = (): boolean => {
+  const [isNativePassInstalled, setIsNativePassInstalled] = useState(false)
+
+  useEffect(() => {
+    const checkNativePassInstallation = async (): Promise<void> => {
+      const canOpenApp = await Linking.canOpenURL('cozypass://')
+      setIsNativePassInstalled(canOpenApp)
+    }
+    void checkNativePassInstallation()
```

Comment:
I would not put this in a `componentDidMount` effect. I estimate it would be a common scenario to install CozyPass whild the Flagship app is still running. So `isNativePassInstalled` would be wrong after that event.

I think it would be better to execute this logic every time we try to open cozy-pass-web

Reply:
I didn't have in mind that you should also inject this in the webview. Maybe we should find a way to inject it only if the rendered slug is cozy-pass-web, so we would not have to do the check everytime we open a cozy-app

Reply:
Agree. It must be solved with what I did for https://github.com/cozy/cozy-flagship-app/pull/705#discussion_r1155089713